### DISCUSSION TITLE
Super slow downloads through VPN. Any Ideas?

### DISCUSSION THREAD

**Discussion by exzacklyright** [transmission/transmission#3279] [Q&A]
- settings.json file

https://i.imgur.com/6uhQEK4.png

- Comparison between transmission and desktop qBittorrent with the same PIA server / torrent
https://imgur.com/nEYp4UK

I'm not sure if the port needs to be opened or not .. .does that affect peers connecting?https://i.imgur.com/nNkxDYT.png

**Comment by devilopa** [comment]
You could try disabling uTP and see what happens. If your VPN gateway is heavily used, uTP could cut down on its speed. Although I am not 100% positive on that one. and i don't even know if qbittorrent has uTP enabled by default.
